### DISCUSSION TITLE
docker-compose --profile [auto] up --build error. How to fix it?

### DISCUSSION THREAD

**Discussion by lancewz** [AbdBarho/stable-diffusion-webui-docker#471] [Q&A]
Setup by docker, but following command run fail.
docker-compose --profile auto up --build

Error information as following(I also try "invoke" profile, but get the similar error.):

 => ERROR [invoke stage-1 6/9] RUN --mount=type=cache,target=/root/.cache/pip   git reset --hard 4463124bddd221c333d4c7  0.4s
------
 > [invoke stage-1 6/9] RUN --mount=type=cache,target=/root/.cache/pip   git reset --hard 4463124bddd221c333d4c70e73aa2949ad35453d &&   pip install -e .:
#0 0.355 HEAD is now at 4463124b feat(nodes): mark ImageField properties required, add docs
#0 0.365 Error processing line 1 of /usr/local/lib/python3.10/site-packages/distutils-precedence.pth:
#0 0.365 
#0 0.371   Traceback (most recent call last):
#0 0.371     File "/usr/local/lib/python3.10/site.py", line 186, in addpackage
#0 0.371       exec(line)
#0 0.371     File "<string>", line 1, in <module>
#0 0.371   AttributeError: module '_distutils_hack' has no attribute 'add_shim'
#0 0.371 
#0 0.371 Remainder of file ignored
#0 0.372 Traceback (most recent call last):
#0 0.372   File "/usr/local/bin/pip", line 5, in <module>
#0 0.372     from pip._internal.cli.main import main
#0 0.372 ModuleNotFoundError: No module named 'pip._internal.cli.main'
------
failed to solve: process "/bin/sh -c git reset --hard 4463124bddd221c333d4c70e73aa2949ad35453d &&   pip install -e ." did not complete successfully: exit code: 1

**Comment by AbdBarho** [comment]
you maybe have an outdated python image, try doing `docker pull python:3.10.9-slim` and run again.
